src/js/components/award/AwardInfo.jsx

```diff
@@ -0,0 +1,64 @@
+/**
+  * AwardInfoContainer.jsx
+  * Created by Emily Gullo 01/20/2017
+  **/
+
+import React from 'react';
+import moment from 'moment';
+
+import SummaryBar from 'components/award/SummaryBar';
+import AwardInfoBar from 'components/award/AwardInfoBar';
+
+const propTypes = {
+    selectedAward: React.PropTypes.object
+};
+
+export default class AwardInfo extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.getStatus = this.getStatus.bind(this);
+    }
+
+    componentWillReceiveProps(nextProps) {
+        if (nextProps.selectedAward !== this.props.selectedAward) {
+            this.getStatus(nextProps.selectedAward);
+        }
+    }
+
+    getStatus(award) {
```

Comment:
Also also, the `parentAwardId` if/else logic in the `SummaryBar` component, I don't know if I'm just getting too nitty gritty, but do you think it's too much logic in the render?